console/src/components/Services/Data/DataActions.js

```diff
@@ -459,7 +459,18 @@ const fetchFunctionInit = (schema = null) => (dispatch, getState) => {
   );
 };
 
-const updateCurrentSchema = (schemaName, redirect = true) => dispatch => {
+const updateCurrentSchema = (
+  schemaName,
+  redirect = true,
+  schemaList = []
+) => dispatch => {
+  if (
+    schemaList.length &&
+    !schemaList.map(s => s.schema_name).includes(schemaName)
```

Comment:
can we simplify this a bit? this is constructing an extra array and full array iteration is happening in all scenarios.
try this 
```jsx
schemaList.find(s=>s.schema_name===schemaName)!==undefined
```